pkg/config/config.go

```diff
@@ -22,17 +22,31 @@ import (
 	"bytes"
 	"time"
 
+	"github.com/golang/glog"
+	"github.com/yaacov/observer/observer"
+
 	"github.com/openshift/autoheal/pkg/apis/autoheal"
 )
 
 // Config is a read only view of the configuration of the auto-heal service.
 //
 type Config struct {
-	awx        *AWXConfig
-	throttling *ThrottlingConfig
-	rules      []*autoheal.HealingRule
+	awx                 *AWXConfig
+	throttling          *ThrottlingConfig
+	rules               []*autoheal.HealingRule
+	configFilesObserver *observer.Observer
 }
 
+// ChangeEvent contains config change event info
+//
+type ChangeEvent struct {
+	Info string
```

Comment:
Can we make `Info` private and, if we want to print it, add a `String` method?

```go
type ChangeEvent struct {
  info string
}

func (e *ChangeEvent) String() {
  return e.info
}
```

That way the user of the `ChangeEvent` type doesn't need to know (and depend on) the fact that it has an `Info` field.